PowerSync/build.gradle.kts

```diff
@@ -47,12 +49,92 @@ skie {
 }
 
 kmmbridge {
-    mavenPublishArtifacts()
+    artifactManager.set(SonatypePortalPublishArtifactManager(project, null, null, null))
+    artifactManager.finalizeValue()
     spm()
-    versionManager.set(TimestampVersionManager)
 }
 
+// We need this so that when a user includes the package in XCode they are able to
+// import the package using Github
 if (System.getenv().containsKey("CI")) {
     // Setup github publishing based on GitHub action variables
     addGithubPackagesRepository()
+}
+
+// This is required for KMMBridge zip to be uploaded to Sonatype (Maven Central)
+// and needs to be in this file as it requires access to import co.touchlab.faktory
+// which is only available here
+class SonatypePortalPublishArtifactManager(
```

Comment:
I'm not sure if you know of another way to import a file into Gradle that can be used? I've tried a few different approaches to no avail. It seems like the only way to accomplish this would require creating a plugin based on my findings, although I might be missing something.